forte/data/ontology/top.py

```diff
@@ -1207,12 +1229,149 @@ def __init__(
         )
 
 
+class Payload(Entry):
+    """
+    A payload class that holds data cache of one modality and its data source uri.
+
+    Args:
+        pack: The container that this `Payload` will
+            be added to.
+        modality: modality of the payload such as text, audio and image.
+        payload_idx: the index of the payload in the DataPack's
+            image payload list of the same modality. For example, if we
+            instantiate a ``TextPayload`` inherited from ``Payload``, we assign
+            the payload index in DataPack's text payload list.
+        uri: universal resource identifier of the data source. Defaults to None.
+
+    Raises:
+        ValueError: raised when the modality is not supported.
+    """
+
+    def __init__(
+        self,
+        pack: PackType,
+        payload_idx: int = 0,
+        uri: Optional[str] = None,
+    ):
+        from ft.onto.base_ontology import (  # pylint: disable=import-outside-toplevel
+            TextPayload,
+            AudioPayload,
+            ImagePayload,
+        )
+
+        # since we cannot pass different modality from generated ontology, and
+        # we don't want to import base ontology in the header of the file
+        # we import it here.
```

Comment:
This is not a solution. We cannot import items that are generated back to the source code. This doesn't make sense.